sklearn/mixture/bayesian_mixture.py

```diff
@@ -0,0 +1,651 @@
+
+    alpha_prior : float, optional.
+        The user-provided alpha prior parameter of the Dirichlet distribution.
+        If is None, the alpha prior is set to `1. / n_components`.
+
+    beta_prior : float, optional.
+        The user-provided beta prior parameter of the Gaussian
+        distribution. If it is None, beta prior is set to 1.
+
+    mean_prior : array-like, shape (`n_features`,), optional
+        The user-provided mean prior of the Gaussian distribution.
+        If it is None, the mean prior is set to the mean of X.
+
+    nu_prior : float, optional.
+        The user-provided nu prior parameter of the covariance distribution.
+        If it is None, the nu prior is set to `n_features`.
+
+    covariance_prior : float or array-like, optional
+        The user-provided covariance prior of the covariance distribution.
+        If it is None, the covariance prior is initialized using the covariance
+        of X. The shape depends on `covariance_type`::
+            (`n_features`, `n_features`) if 'full',
+            (`n_features`, `n_features`) if 'tied',
+            (`n_features`)               if 'diag',
+            float                        if 'spherical'
+
+    random_state: RandomState or an int seed, defaults to None.
+        A random number generator instance.
+
+    warm_start : bool, default to False.
+        If 'warm_start' is True, the solution of the last fitting is used as
+        initialization for the next call of fit(). This can speed up
+        convergence when fit is called several time on similar problems.
+
+    verbose : int, default to 0.
+        Enable verbose output. If 1 then it prints the current
+        initialization and each iteration step. If greater than 1 then
+        it prints also the log probability and the time needed
+        for each step.
+
+    Attributes
+    ----------
+    weights_ : array-like, shape (`n_components`,)
+        The weights of each mixture components.
+
+    means_ : array-like, shape (`n_components`, `n_features`)
+        The mean of each mixture component.
+
+    covariances_ : array-like
+        The covariance of each mixture component.
+        The shape depends on `covariance_type`::
+            (n_components,)                        if 'spherical',
+            (n_features, n_features)               if 'tied',
+            (n_components, n_features)             if 'diag',
+            (n_components, n_features, n_features) if 'full'
+
+    precisions_ : array-like
+        The precision matrices for each component in the mixture. A precision
+        matrix is the inverse of a covariance matrix. A covariance matrix is
+        symmetric positive definite so the mixture of Gaussian can be
+        equivalently parameterized by the precision matrices. Storing the
+        precision matrices instead of the covariance matrices makes it more
+        efficient to compute the log-likelihood of new samples at test time.
```

Comment:
In fact we give precisions_ to accelerate of the function predict which is in fact the log-likelihood.